service/api/types.go

```diff
@@ -73,6 +73,16 @@ type Breakpoint struct {
 	TotalHitCount uint64 `json:"totalHitCount"`
 }
 
+// ExecutionPoint addresses a location at which process execution will jump to.
+type ExecutionPoint struct {
```

Comment:
This type is basically the same thing as an api.Location. Given that it's constructed starting from an api.Location I think we should pass in the api.Location directly.

Alternatively just pass the address.